ch_pipeline/core/io.py

```diff
@@ -473,3 +472,52 @@ def next(self, files):
         )
 
         return sorted(list(new_files))
+
+class SaveGaltAutoCorrelation(task.SingleTask):
```

Comment:
Maybe this is a bit nit-picky, but I would suggest renaming this to something like `ExtractGaltAutocorrelation`and moving it out of `io` (since it only performs io if you choose to save the output, same as any task). I'm not sure where would be the best place for it... Maybe `analysis.calibration` since I assume this is meant for noise source analysis. `analysis.beam` could also work.